### ISSUE TITLE
Android crashes with TypeError · undefined is not a function (evaluating 'babelHelpers.readOnlyError("EVENT_SUBSCRIPTIONS")') 

### ISSUE THREAD

**Issue by ruimagalhaes** [transistorsoft/react-native-background-geolocation#552]
* Platform: Android

I have an app deployed on production with the plugin and bugsnag is recording multiple app crashed related with the plugin. 

My plugin config:

```
{
        reset: true,
        desiredAccuracy: 0,
        distanceFilter: 5,
        stopTimeout: 1,
        debug: false,
        logLevel: BackgroundGeolocation.LOG_LEVEL_OFF,
        stopOnTerminate: false, 
        startOnBoot: true
      }
```

The recorded error:

```
TypeErrornode_modules/react-native-background-geolocation/NativeModule.js:193
undefined is not a function (evaluating 'babelHelpers.readOnlyError("EVENT_SUBSCRIPTIONS")')
```

![screen shot 2018-08-28 at 14 25 15](https://user-images.githubusercontent.com/888930/44725829-3bea5d00-aace-11e8-895c-704a5a413373.png)

Any idea of what may be causing the issue?

Thanks!

**Comment by christocracy** [comment]
## Your Environment
* Plugin version:
* Platform: iOS or Android
* OS version:
* Device manufacturer / model:
* React Native version (`react-native -v`):
* Plugin config

**Comment by ruimagalhaes** [comment]
Plugin version: 2.12.2
    Platform: Android
    OS version: 8.0.0
    Device manufacturer / model: Samsung SM-G960U
    React Native version (react-native -v): 0.56.0

**Comment by christocracy** [comment]
This was reported [in July](https://github.com/transistorsoft/react-native-background-geolocation/pull/523) and the fix applied to latest version `2.13.1`
